pombola/core/models.py

```diff
@@ -1909,3 +1909,18 @@ def raw_query_with_prefetch(query_model, query, params, fields_prefetches):
             if related_object:
                 setattr(o, field, related_object)
     return objects
+
+
+class OrganisationHistory(ModelBase):
+    old_organisation = models.ForeignKey('Organisation', null=False, related_name='org_history_old')
+    new_organisation = models.ForeignKey('Organisation', null=True, blank=True, related_name='org_history_new')
```

Comment:
I don't think this should be allowed to be blank. What was your thinking here? Perhaps I'm missing something